atom/browser/api/atom_api_window.cc

```diff
@@ -786,6 +786,18 @@ bool Window::IsVisibleOnAllWorkspaces() {
   return window_->IsVisibleOnAllWorkspaces();
 }
 
+void Window::SetVibrancy(v8::Local<v8::Value> value, mate::Arguments* args) {
+  std::string type;
```

Comment:
> Is that an acceptable behavior?

Yeah, many other Electron APIs behave like this, I think this is something to improve upon and provide more consistent errors/checks but we'd do that module by module for Electron 2.0.